items/boosters.json

```diff
@@ -0,0 +1,42 @@
+{
+	"684729784": {
+		"name": "Stamina Enhancement",
+		"desc": "Increases all Helldivers' stamina capacity and recovery.",
+		"lore": "Increases total stamina and recovery rate by roughly 30%. This booster let's our endurance reaches new heights, allowing us to push through fatigue and keep fighting at peak performance. Whether it's a marathon sprint across hostile terrain or a prolonged firefight, the Stamina Enhancement booster ensures we stay in the fight longer and hit harder."
```

Comment:
I think this could be called `details` (as that's where it's shown in the game), though I'm open for discussion on this one

Reply:
I'm mistaken, it's not shown in game so `lore` is fine

Reply:
Ended up not being real, removed